subsys/bluetooth/mesh/light_ctl_srv.c

```diff
@@ -316,9 +320,9 @@ static void default_set(struct bt_mesh_model *model,
 	uint16_t temp = net_buf_simple_pull_le16(buf);
 	uint16_t delta_uv = net_buf_simple_pull_le16(buf);
 
-	if ((temp < BT_MESH_LIGHT_TEMP_RANGE_MIN) ||
-	    (temp > BT_MESH_LIGHT_TEMP_RANGE_MAX)) {
-		goto respond;
```

Comment:
Can remove the `respond` label below now

Reply:
Yes, it's right. I thought that changes from this pull request https://github.com/nrfconnect/sdk-nrf/pull/3310 will affect this part: https://github.com/nrfconnect/sdk-nrf/blob/aadf7e33edd59805542e6f1994d45a91aeab3389/subsys/bluetooth/mesh/light_ctl_srv.c#L312-L348